examples/umbrella_integration/integration.py

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env python3
+import sys
+import argparse
+import numpy as np
+import matplotlib.pyplot as plt
+
+import hoomd
+hoomd.context.initialize("")
```

Comment:
Let's move this line either at the beginning of `generate_context` or into `main` before `method.run`

Reply:
I agree. The line was misplaced.
I moved it into the context generation. In case we run on different nodes we need to set this up later.